sympy/geometry/tests/test_line.py

```diff
@@ -53,6 +53,14 @@ def test_angle_between():
                                 Line3D(Point3D(0, 0, 0), Point3D(5, 0, 0))), acos(sqrt(3) / 3)
 
 
+def test_closing_angle():
```

Comment:
Nothing to worry about I think. I took this from the latest version of test_line.py of master as I had some version issues with my local copy of this file.